releasenotes/notes/deprecate-constant-9bc432d6647766f3.yaml

```
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    ``ddtrace.contrib.grpc.constants.GRPC_PORT_KEY`` is deprecated. Use ``ddtrace.ext.net.TARGET_PORT`` instead.
```

We should use third person present tenses to describe changes in release notes [guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#release-note-style-guidelines).

```suggestion
    grpc: deprecates ``ddtrace.contrib.grpc.constants.GRPC_PORT_KEY``. Use ``ddtrace.ext.net.TARGET_PORT`` instead.
```

---

Teachable moment about cargo-culting the wording instead of skimming over that part of release note docs 😅